.clang-format

```diff
@@ -0,0 +1,22 @@
+---
+# We'll use defaults from the LLVM style, but with 4 columns indentation.
```

Comment:
How to run this?
While this is not built-in automated check in the project, at least, we should add some documentation on how to locally run this formatter or check the format.
The truth is that without ci/cd check, the formatting won´t be ensured for every contribution. And if that is not the case... this is yet not very useful.

Reply:
.clang_format removed!